scripts/download

```diff
@@ -8,17 +8,20 @@ RUNC_VERSION=v1.0.0-rc94
 ROOT_VERSION=v0.8.1
 TRAEFIK_VERSION=9.18.2  # appVersion: 2.4.8
 CHARTS_DIR=build/static/charts
+RUNC_DIR=build/src/github.com/opencontainers/runc
 DATA_DIR=build/data
 export TZ=UTC
 
 umask 022
+rm -rf ${CHARTS_DIR}
+rm -rf ${RUNC_DIR}
```

Comment:
fixes errors when running the download and generate tasks after switching between branches